plugins/flyte.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: flyte
+spec:
+  version: "v0.5.12"
+  homepage: https://github.com/lyft/flyte
+  shortDescription: kubectl-flyte is an command line tool for flyte that can be used as an extension to kubectl.
+  description: |+2
+      kubectl-flyte is an command line tool for flyte that can be used as an extension to kubectl.
```

Comment:
This doesn't describe anything to me really. Can you explain a but what Flyte does?

Please read other plugin descriptions that have an unfamiliar project name like this. See how they explain